src/grpc/helium_stream_poc_impl.erl

```diff
@@ -0,0 +1,163 @@
+-module(helium_stream_poc_impl).
+
+-include("../../include/sibyl.hrl").
```

Comment:
do these header files in the `include/` dir need relative paths? i thought rebar knew to look for `.hrl` files in a root `include/` directory so these could just be `-include("sibyl.hrl").`

Reply:
They should but yet it was failing to be found....may have been a rebar bug at the time